app/views/reports/new.html.erb

```diff
@@ -2,17 +2,24 @@
 
 <%= simple_form_for @report do |form| %>
   <%= form.association :recipient, label_method: :first_name, value_method: :id %>
-    <% KeyTrait::CATEGORIES.each do |key_trait| %>
-      <h2><%= key_trait.titleize %></h2>
+    <% @skills_data.each do |key_trait, skill_groups| %>
```

Comment:
So the second parameter is the first nested level of the JSON file? I didn't know we could do this. If this works, brilliant! However, I think the name @skills_data is a little confusing. I think it should be @report_data/@report_keys since it's all-encompassing.